.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
ignore this file?

Reply:
I think it's helpful to all the vscode users so I add it here.
It will help Go extension to load the local env file (just as instructed in the README) while running tests in vscode.
Should I ignore this?